src/view/show_mode/editor/show_ui_widgets/cue_control.py

```diff
@@ -71,12 +86,23 @@ def construct_widget(self, parent: QWidget | None, enabled: bool):
             item.annotated_data = cue
             cue_list.addItem(item)
         cue_list.setEnabled(enabled)
-        cue_list.setMinimumHeight(100)
+        cue_list.setMinimumHeight(300)
```

Comment:
i changed the height (for understanding the ui), and found the name of the filter below. It does not float (should it?) below the other ui elements of the filter.

Reply:
No, I think it is fine if it does not float but is visible.